src/gotoDefinition.ts

```diff
@@ -48,4 +53,9 @@ export const registerGotoDefinition = () => {
             return [{ range, targetRange: range, uri, targetUri: uri }]
         },
     })
+    setFindReferenceButtonVisibility()
```

Comment:
Would be better to move it to the end of https://github.com/zardoy/vetur-extended/blob/main/src/findReferences.ts

Reply:
Done